### DISCUSSION TITLE
Caching for GraphQL at EDGE (Help needed if it sounds interesting)

### DISCUSSION THREAD

**Discussion by bishwenduk029** [urql-graphql/urql#1310] [Show and tell]
Urql library seems to have a solution for caching GraphQL responses and also the API is a delight to use . I was recently very inspired by frameworks like NextJs and [Flareact](https://flareact.com/).

In fact I have opened a new PR for rather an opinionated way of extending Flareact to also support GraphQL execution at Edge([Flqreact](https://github.com/bishwenduk029/flqreact)). Please find the details of the advantaged and motivations summarised in this [PR](https://github.com/flareact/flareact/pull/103). Please feel free to provide your feedback on the same.

One of the challenges I am facing is implementing a Caching solution for the same. I am Urql for querying at edge. I am also working on creating a custom Exchange to interact with Cloudflare's Cache API but I am need some guidance and help from you if you would be interested.

Thanks.

**Comment by LeoBorai** [comment]
Hi @bishwenduk029, I see this question is a bit old now, but I wanted to know how it went for you? Im also using URQL for a project hosted on Cloudflare Pages and I'm having issues with `cache` options in `fetchOptions` when using my URQL Client.

Thanks in advance.
